src/websharkView.ts

```diff
@@ -217,6 +221,7 @@ export class WebsharkViewReadonlyEditorProvider implements vscode.CustomReadonly
     }
     resolveCustomEditor(document: WebsharkViewCustomDocument, webviewPanel: vscode.WebviewPanel, token: vscode.CancellationToken): Thenable<void> | void {
         console.log(`WebsharkViewReadonlyEditorProvider resolveCustomEditor(document.uri=${document.uri.toString()}, webviewPanel:${webviewPanel}) called`);
+        const _wiresharkProfile = <string>(vscode.workspace.getConfiguration().get("vsc-webshark.wiresharkProfile"));
```

Comment:
please add to package.json ... contributes ... configuration ...  properties as well so that the vscode ui offers this new setting